### DISCUSSION TITLE
Meeting: ecatalogue Working Group - 25 April 2023

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#821] [MEETINGS]
Agenda: EMu Hour
## ​​ecatalogue Working Group [part ~1]
-- Vocabularies for the following lookup tables in test
[Sex](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=1659415673)
- - Field Help Definition: The sex of the specimen(s)
- - How deal with counts?
- - Separate lines/columns in Preps/Parts table
* * * What values should be in this field?:
Female
Male
Gynandromorph
Monoecious* = both male & female in the same individual (botany)
Rabbit hole: cryptogams vs flowering plants may have slightly different spelling for terminology
Hermaphrodite= both male & female in the same individual (zoology)
Indeterminate = e.g. specimen has been examined, but sex is not knowable/determinable
Not recorded = [pending semaphorant]
Sterile
[leave blank if not relevant to data – e.g. mineral, nest]
Repatriation values:
probable male
probable female 
possible male
possible female
unknown
Other notes
For terms like ‘super queen’, can pull from Semaphoront field 
In a record for a jar/lot of males + females, could record “Male” on one line, and “Female” on another line (+/- a count/estimate in the separate count field for each row)
For abiotic specimens (nests, trace fossils), addressing whether to leave sex blank?
E.g. for Insects catalogue records where Semaphoront field is required, what to do (RB to share Insects data/example)
For more complex mating types (Botany) – use an available basic term in ‘Sex’ + more detail in Remarks (or propose new terms where needed)

[Lifestage](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=145053962)
- - Adult
Embryo(s)
Egg(s)
Brood
Larva
Pupa
Nymph
Subadult
subimago
Super male 
Queen - alate (female only)
Queen - dealate (female only)
Soldier (female only)
Worker (female only)
Hatchling
Fetus
Gravid
Protoconch
Juvenile
unknown

Field Help Definition:Lifecycle stages from fertilization to adult/maturity.
Botany – 
Overlap with Part/Repro. Organ field?
For flowering plants – possibly “seedling”/”sapling” – tbd

# Homework for 3 May 2023 meeting

[Measurement Taken](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=496435033)
Field Help Definition:
And define data being recorded
[Measurement Kind](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=178476466)
Field Help Definition:
And define data being recorded
Preps and Parts (and indicators’ logic)
Field Help Definition:
Other # Source (Catalog Tab)
Field Help Definition
[From Group Testing](https://docs.google.com/spreadsheets/d/1MUg92tIPsuuhkWAQOM4iIB4nwJV_z2oM4oCq4b7PB04/edit?pli=1#gid=1658218035)

**Comment by fmjjones** [comment]
Potential definition for PheStage_tab: : the series of stages in form and functional activity through which an organism passes between successive recurrences of a specified primary stage. from Mearriam-Webster https://www.merriam-webster.com/dictionary/life%20cycle

**Comment by fmjjones** [comment]
Monoecious and Dioecious are Taxon level descriptors and belong in the Taxonomy module. Use Male & Female for catalogue level descriptions when necessary in Botany. [WG]

**Comment by rbaquiran** [comment]
Brood can be removed from lifestage if no one besides Insects uses it or is interested in it. Insect data cleanup has removed this and replaced with nymph or larva where appropriate.
